test/unit/puppetlabs/puppetserver/ruby/http_client_test.clj

```diff
@@ -207,7 +225,10 @@
             (is (true? false) "Expected HTTP connection to HTTPS port to fail")
             (catch EvalFailedException e
               (is (instance? HttpClientException (.. e getCause)))
-              (is (ssl-connection-exception? (.. e getCause getCause)))))))
+              (is (ssl-connection-exception? (.. e getCause getCause)))))
+          (.runScriptlet sc "Puppet::Server::HttpClient.terminate")
+          (.terminate sc)))
```

Comment:
Would probably be best to DRY up this pattern by creating a utility function that does the two terminate calls and then re-using that across all of the tests.